workspace/__lib__/xod/stream/accumulate-string/patch.xodp

```diff
@@ -31,6 +33,7 @@
       "type": "xod/patch-nodes/not-implemented-in-xod"
```

Comment:
A comment for `CAP`: “Capacity of the string buffer. Defines the maximum length. Must be a constant…”